cmd/oras/root/manifest/push.go

```diff
@@ -150,6 +150,8 @@ func pushManifest(cmd *cobra.Command, opts pushOptions) error {
 	// prepare manifest descriptor
 	desc := content.NewDescriptorFromBytes(mediaType, contentBytes)
 
+	displayStatus, displayMetadata := display.NewManifestPushHandler(opts.Printer, opts.OutputDescriptor, opts.Pretty.Pretty, desc, opts.Target)
```

Comment:
```suggestion
	statusHandler, metadataHandler := display.NewManifestPushHandler(opts.Printer, opts.OutputDescriptor, opts.Pretty.Pretty, desc, opts.Target)
```

and wherever they are used.

That was a suggestion on PR and I like those names better.

Reply:
Renamed.